tests/messages/test_traction_qc_message.py

```diff
@@ -79,7 +83,7 @@ def test_can_generate_correct_payload(self, valid_traction_qc_message):
                             "shearing_qc_comments": "Comments",
                             "date_submitted": datetime.now().timestamp() * 1000,
```

Comment:
datetime.now without timezone is locale dependent I suppose. Does this matter?

Reply:
This is fine in a test.  I would be concerned if it was part of the code creating messages or logs to be sent out.  I'll update it in the test anyway to avoid misleading someone who sees this and thinks that's how you create the timestamp in messages.